platforms/core-runtime/launcher/build.gradle.kts

```diff
@@ -38,6 +38,8 @@ dependencies {
     api(projects.serviceLookup)
     api(projects.serviceProvider)
     api(projects.snapshots)
+    api(projects.snapshotsWorker)
+    api(projects.requestHandlerWorker)
```

Comment:
This dependency on the RequestHandler infrastructure should not be here. 

The launcher should not depend on worker implementation classes.